lighthouse-cli/test/cli/run-test.js

```diff
@@ -23,7 +23,7 @@ const getFlags = require('../../cli-flags').getFlags;
 
 describe('CLI run', function() {
   it('runLighthouse completes a LH round trip', () => {
-    const url = 'chrome://version';
+    const url = 'chrome://version/';
```

Comment:
hang on, I just realized the comment around this. Are we doing something wrong here?

just tried this out in console, seems like a bad result if everything gets a trailing slash now
```
const URL = require('whatwg-url').URL
console.log(new URL('chrome://version').href) // 'chrome://version'
```

Reply:
Okay looks like special urls had a trailing slash for a little bit in the spec but it was changed.

https://github.com/w3c/web-platform-tests/pull/4586
https://github.com/whatwg/url/pull/213
https://github.com/jsdom/whatwg-url/commit/9ec56188362b5775c53072ba0cfeb437c1e02709

However Chrome doesn't yet implement this change. As a result, it adds the trailing slash when it delivers the canonicalized URL over the devtools protocol. In the future this'll probably change, but for now we'll have to normalize between the spec-violating Chrome and the correct whatwg-url.

Doesn't look like there's a crbug for this, though there's a sprinkling of other issues documenting Chrome's URL failures on webplatformtests.

exciting. 🤒